package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mbed-connector-api",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
Not sure how other node packages usually handle this, but I think I'd prefer to create a dedicated commit to bump this number instead of merging this in with these changes.